pkg/core/localize/locales/en/cmd/acl.en.toml

```diff
@@ -85,6 +85,9 @@ one = 'Set the resource type to cluster'
 [kafka.acl.common.flag.prefix.description]
 one = 'Determine if the resource should be exact match or prefix'
 
+[kafka.acl.common.flag.patterntypes.description]
+one = 'Determine if the resource should be exact match, prefix or any {{.Types}}'
```

Comment:
The flag description looks odd

```
Determine if the resource should be exact match, prefix or any [any literal prefix] (default "literal")
```

Should we keep options in description only?

Reply:
It is good to give list of values in canonical form. Will remove them from description